File: 1627-last-moment-before-all-ants-fall-out-of-a-plank/1627-last-moment-before-all-ants-fall-out-of-a-plank.py
class Solution:
    def getLastMoment(self, n: int, left: List[int], right: List[int]) -> int:
        ans = 0
        for val in left:
            ans = max(ans,val)
        for val in right:
            ans = max(n-val, ans)
        return ans

File: 1627-last-moment-before-all-ants-fall-out-of-a-plank/1627-last-moment-before-all-ants-fall-out-of-a-plank.java
class Solution {
    public int getLastMoment(int n, int[] left, int[] right) {
        int min = Integer.MAX_VALUE;
        int max = Integer.MIN_VALUE;
        for(int i: left){
            max = Math.max(i,max);
        }
        for(int i: right){
            min = Math.min(i,min);
        }
        return Math.max(n - min,max);
    }
}